Sources/PointFreeTestSupport/MockEpisode.swift

```diff
@@ -0,0 +1,99 @@
+@testable import EpisodeTranscripts
+
+extension Episode {
+  public static let mock = proofInFunctions
+}
+
+private let proofInFunctions = Episode(
```

Comment:
this is the mock episode we'll use for tests. I wanted to keep this PR small, but soon I want to update this mock episode to be a better example of all the different things that can show up in an episode.